external-crates/move/crates/move-analyzer/src/inlay_hints.rs

```diff
@@ -32,13 +32,9 @@ pub fn on_inlay_hint_request(context: &Context, request: &Request) {
         vec![]
     };
 
-    let response = lsp_server::Response::new_ok(request.id.clone(), hints);
```

Comment:
Unrelated, but changed qualified enum type to imported one all over for consistency